lib/experimental/Widgets/Content/CopyLabel/index.tsx

```diff
@@ -0,0 +1,33 @@
+import { Icon } from "@/components/Utilities/Icon"
+import { LayersFront } from "@/icons"
+import { forwardRef } from "react"
+interface CopyLabelType {
+  text: string
+}
+
+export const CopyLabel = forwardRef<HTMLDivElement, CopyLabelType>(
+  ({ text }, ref) => {
+    const copyHandler = async () => {
+      try {
+        await navigator.clipboard.writeText(text)
+      } catch (error) {
+        void error
+      }
+    }
+
+    return (
+      <div
+        ref={ref}
+        className="flex h-8 list-none items-center justify-between rounded-md p-1.5 font-medium text-f1-foreground-secondary hover:cursor-pointer hover:bg-f1-background-secondary"
```

Comment:
I wouldn't set a fixed `height` here. We shuould be able to reach this otherwise. If the icon is taking up too much space, then let's remove `p-1.5` and apply a smaller one?